Makefile

```diff
@@ -10,9 +10,12 @@ USER_AGENT=${GIT_REPO}/${PACKAGE_VERSION}
 
 OPENAPI_IMAGE_TAG=v7.4.0
 OPENAPI_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_IMAGE_TAG}
+GOSWAGGER_IMAGE_TAG=v0.31.0
+GOSWAGGER_IMAGE=ghcr.io/go-swagger/go-swagger:${GOSWAGGER_IMAGE_TAG}
 CRI=docker # nerdctl
 CRI_COMMAND_BASE=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} $(DOCKER_EXTRA_ARGS)
-OPENAPI_GENERATOR=${CRI_COMMAND_BASE} -v $(CURDIR):/local ${OPENAPI_IMAGE}
+GOSWAGGER=${CRI_COMMAND_BASE} -v $(CURDIR):/local -w /local ${GOSWAGGER_IMAGE}
+OPENAPI_GENERATOR=${CRI_COMMAND_BASE} -v $(CURDIR):/local -w /local ${OPENAPI_IMAGE}
```

Comment:
I added `-w /local` because the argument references sometimes lead with `/local` and sometimes didn't. As I moved between local tools and docker tools the `/local` part of the argument list needed to be removed. Working out of `/local` means we don't need to lead with `/local`, but it doesn't hurt to have it.